modules/network-observability-lokistack-create.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-lokistack-create_{context}"]
+
+= Create a LokiStack custom resource. 
+
+.Procedure
+
+. Navigate to *Operators* -> *Installed Operators*. In the Details, under *Provided APIs*, select `LokiStack` and click *Create LokiStack*. 
+. Esure the following fields are specified in either *Form View* or *YAML view*:
++
+[source,yaml]
+----
+  apiVersion: loki.grafana.com/v1
+  kind: LokiStack
+  metadata:
+    name: loki
+    namespace: netobserv
+  spec:
+    size: 1x.small
+    storage:
+      schemas:
+      - version: v12
+        effectiveDate: '2022-06-01'
+      secret:
+        name: loki-s3
+        type: s3
+    storageClassName: gp3
+    tenants:
+      mode: openshift-network
+----
++
+[IMPORTANT]
+====
+You must not reuse the same LokiStack that is used for cluster logging.
+====
+
+== Configuring LokiStack Rate Limits
+Two notable LokiStack specifications, `ingestionRate` and `ingestionBurstSize`, might have an impact on the flow processing pipeline. If these rate limits are reached, Loki returns an error when `flowlogs-pipeline` tries to send batches. 
```

Comment:
Should we call out `maxGlobalStreamsPerTenant` here as well? cc @jotak @mariomac @jpinsonneau

Reply:
yes it makes sense @nathan-weinberg. It's already in the yaml below

Reply:
@jpinsonneau @nathan-weinberg is there anything to describe differently about `maxGlobalStreansPerTenant`? Is the customer missing any other info about setting these?

Reply:
I guess currently there are few details, just "If these limits are reached, Loki returns an error when `flowlogs-pipeline` tries to send batches." But are there distinct things to note about these configs?

Reply:
loki-operator `maxGlobalStreansPerTenant` configure loki `max_global_streams_per_user` that I also documented upstream here https://github.com/netobserv/documents/blob/main/loki_config.md#maximum-active-stream-limit-exceeded

Here is the [official loki doc](https://grafana.com/docs/loki/latest/configuration/):
```
# Maximum number of active streams per user, across the cluster. 0 to disable.
# When the global limit is enabled, each ingester is configured with a dynamic
# local limit based on the replication factor and the current number of healthy
# ingesters, and is kept updated whenever the number of ingesters change.
# CLI flag: -ingester.max-global-streams-per-user
[max_global_streams_per_user: <int> | default = 5000]
```

I'm not sure we should go too much in details in the installation section as it depends on various parameters / cluster size / expected flows. Can we keep the details in troubleshooting ?

Reply:
Looping in @xenolinux for consideration in Troubleshooting.

Reply:
Ack!

Reply:
Removing this from Installing, as Servesha has incorporated into Troubleshooting area: https://github.com/openshift/openshift-docs/blob/9d3f2d9087e793197fecf0fc1101725f98bdf15b/modules/troubleshooting-network-observability-lokistack-rate-limits.adoc